recipe-server/client/control_new/less/pages/gateway.less

```diff
@@ -0,0 +1,31 @@
+.page-gateway {
+  .anticon {
+    display: block;
+    font-size: 3em;
+    margin-bottom: 0.5em;
+    text-align: center;
+  }
+
+  .gw-col {
+    background: #E1F4CB;
```

Comment:
Based on [this blog post](https://medium.com/@GeoffMiller/how-to-customize-ant-design-with-react-webpack-the-missing-guide-c6430f2db10f), I think it'd be better to make color variables its own PR. Looks like we have to play with Webpack to get it to play nice with Less.